terraform/datadog/router.tf

```diff
@@ -0,0 +1,165 @@
+resource "datadog_monitor" "router" {
+  name = "${format("%s router hosts", var.env)}"
+  type = "service check"
+  message = "Missing router hosts in environment {{host.environment}}. Notify: @the-multi-cloud-paas-team@digital.cabinet-office.gov.uk"
+  escalation_message = "Missing router hosts! Check VM state."
+  no_data_timeframe = "2"
+  query = "${format("'datadog.agent.up'.over('environment:%s','job:router').by('*').last(1).pct_by_status()", var.env)}"
+
+  thresholds {
+    ok = 0
+    warning = 0
+    critical = 10
+  }
+
+  require_full_window = true
+  tags {
+    "environment" = "${var.env}"
+    "job" = "router"
+  }
+}
+
+resource "datadog_timeboard" "gorouter" {
+  title = "${format("%s - GoRouter", var.env)}"
```

Comment:
The naming is different than the runtime difference dashboard. Can we make the same?
Like `title = "${format("%s gorouter", var.env)}"`